metrics/collectors/images/images_collector.py

```diff
@@ -110,6 +110,7 @@ def collect(self):
                             "flavor": flavor,
                             "release": series,
                             "current_or_pending": current_or_pending,
+                            "archictecture": arch,
```

Comment:
```suggestion
                            "architecture": arch,
```

I can fix this before merging!